.eslintrc.js

```diff
@@ -1 +1,20 @@
-module.exports = require('@centreon/eslint-config-centreon');
\ No newline at end of file
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  plugins: ["@typescript-eslint"],
+  extends:  [
+    '@centreon/eslint-config-centreon'
+  ],
+  settings: {
+    'import/resolver': {
+      alias: {
+        map: [
+          ['@centreon/ui', '@centreon/ui/src'],
+        ],
+        extensions: ['.js', '.jsx',]
+      }
+    }
+  },
+  rules: {
+    "react/jsx-filename-extension": [2, { "extensions": [".jsx", ".tsx"] }]
+  }
```

Comment:
can we move these rules into eslint-config-centreon?

Reply:
Of course